src/evaluation/liveRec/idScoresBAC.m

```diff
@@ -0,0 +1,70 @@
+function [idLabels, perfs] = idScoresBAC(bbs, labels, onOffsets)
+
+fprintf( '\n\nEvaluate scores...\n\n' );
+idHyps = bbs.blackboard.getData( 'identityHypotheses' );
+idMismatch = getIdScores( idHyps );
+idLabels = sort( fieldnames( idMismatch ) );
+
+% assume blockSize remains constant throughout
+% assume all hypotheses have the same concernsBlocksize_s
+assert( numel(labels) == numel(onOffsets) );
+labelBlockSize_s = idHyps(1).data(1).concernsBlocksize_s;
+%blockAnnotations_list = {};
+types = cell(1, numel( labels ) );
+for il = 1 : numel( labels )
+    if ~isempty(labels{il})
+        types(il) = labels{il}(1);
+        for idl = 1 : numel( idLabels )
+            if strcmp(idLabels{idl}, types{il})
+                idMismatch.(idLabels{idl}).labelIdx = il;
+            end
+        end % idLabels
+    end
+end % labels
+
+labeler = StandaloneMultiEventTypeLabeler( ...
+            'labelBlockSize_s', labelBlockSize_s, ...
+            'types', types );

```

Comment:
If I am not mistaken, you are creating a multinomial labeler here (if numel( types ) > 1), which will produce output different from [-1,1]. But below you assume just that, I believe. Or is the StandaloneMultiEventTypeLabeler behaving differently?

Where IS the StandaloneMultiEventTypeLabeler? ;)